Dockerfile

```diff
@@ -0,0 +1,39 @@
+#*********************************************************************
+# Copyright (c) Intel Corporation 2021
+# SPDX-License-Identifier: Apache-2.0
+#*********************************************************************/
+
+FROM ubuntu:20.04 AS rpc-builder
+
+WORKDIR /
+ARG DEBIAN_FRONTEND=noninteractive
+RUN \
+  apt-get update -y -qq && \
+  apt install -y -qq \
```

Comment:
We removed dependency on needing boost libraries, so I believe those can be removed.